breaking-front/src/pages/PostWrite/PostWrite.styles.js

```diff
@@ -0,0 +1,169 @@
+import Button from 'components/Button/Button';
+import styled, { css } from 'styled-components';
+
+export const PostWriteTitle = styled.h3`
+  font-size: 24px;
+`;
+
+export const PostRadioButton = styled(Button)`
+  ${({ id, radioContorl }) =>
+    id === radioContorl &&
+    css`
+      background-color: ${({ theme }) => theme.blue[400]};
+    `}
+`;
```

Comment:
```suggestion
export const PostRadioButton = styled(Button)`
 background-color: ${({ id, radioControl, theme }) =>
    id === radioContorl && theme.blue[400]};
`;
```

그리고 radioContorl -> radioControl로 오타 수정바랍니다. 😁